### ISSUE TITLE
Проблемы с отзывчивостью интерфейса

### ISSUE THREAD

**Issue by Kommprog** [proninyaroslav/libretrack#35]
Приложение хорошее, но на нескольких девайсах, где оно пробовалось, наблюдаются странные баги: при тапе на любой элемент интерфейса (шапка меню, какая-нибудь кнопка, любой элемент внутри приложения вообще) нажатие происходит _практически_ всегда не с первого раза, а со второго-третьего-четвёртого, будто проблемы у смартфона с тачем (что, конечно, не так) или что-то с областью DPI (как это верно называется - не в курсе).

**Comment by proninyaroslav** [comment]
Это наблюдается только при первом запуске или же последующие тоже?

**Comment by Kommprog** [comment]
> Это наблюдается только при первом запуске или же последующие тоже?

При абсолютно каждом запуске.

**Comment by proninyaroslav** [comment]
Попробуйте в версии 1.5.0. Если повторится, откройте снова.

**Comment by Komprogue** [comment]
> Попробуйте в версии 1.5.0. Если повторится, откройте снова.

Да, теперь действительно этот старый баг с невосприимчивостью касаний ушёл.
